tests/jsTests/indexSpec.js

```diff
@@ -0,0 +1,283 @@
+/**
+ * Suite : Tests for showing error on incorrect field entry
+ * Description : This is a test suite for showing error in case of an invalid field entry
+ */
+describe("Tests for showing error on incorrect field entry", function() {
+	var msg = 'This is a test message for showing error';
+	beforeEach(function() {
+		setFixtures('<div class="field"> \
+				<div class="ui red pointing"></div> \
+			</div>');
+		showError($('.field'), msg);
+	});
+
+	it("should be visible", function() {
+		expect($('.field')).toHaveClass('error');
+		expect($('.ui.red.pointing')).toContainText(msg);
+		expect($('.ui.red.pointing')).toBeVisible();
+	});
+});
+
+/**
+ * Suite : Tests for hiding error on correct field entry
+ * Description : This is a test suite for hiding error in case of an valid field entry
+ */
+describe("Tests for hiding error on correct field entry", function() {
+	var msg = 'This is a test message for not showing error';
+	beforeEach(function() {
+		setFixtures('<div class="field"> \
+				<div class="ui red pointing">' + msg + '</div> \
+			</div>');
+		hideError($('.field'));
+	});
+
+	it("should be visible", function() {
+		expect($('.field')).not.toHaveClass('error');
+		expect($('.ui.red.pointing')).toBeHidden();
+	});
+});
+
+/**
+ * Suite : Tests for not empty field
+ * Description : This is a test suite for checking when field is not empty
+ */
+describe("Tests for not empty field", function() {
+	var msg = 'This is a test message when field is not empty',
+		r = false;
```

Comment:
@MehaKaushik It is already declared. This is one way of declaring variables. 
Reference: 
https://www.w3schools.com/js/tryit.asp?filename=tryjs_variables_multiline
Yes, I am changing the name. 

@kavithaenair Indentation is proper, this is a github issue,